CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ If you can't share your code/replays publicly try to replicate with a smaller sc
 Patches
 =========
 
-Please submit patches by pull request where possible. Patches should add a test to confirm their fix and should not break previously working tests.
+Please submit patches by pull request where possible. Patches should add a test to confirm their fix and should not break previously working tests.   Circle CI automatically runs tests on each pull request so please check https://circleci.com/gh/ggtracker to see the results of those tests.
```

Comment:
I believe the link should be https://circleci.com/gh/ggtracker/sc2reader , not https://circleci.com/gh/ggtracker . I actually can't access the latter link